src/Hazelcast.Net.Tests/Exceptions/RemoteExceptionTests.cs

```diff
@@ -32,13 +32,13 @@ public void Constructors()
             _ = new RemoteException();
             _ = new RemoteException("exception");
             _ = new RemoteException("exception", new Exception("bang"));
-            _ = new RemoteException(RemoteError.Undefined, true);
-            _ = new RemoteException(RemoteError.Undefined, "exception");
-            _ = new RemoteException(RemoteError.Undefined, "exception", true);
-            _ = new RemoteException(RemoteError.Undefined, new Exception("bang"));
-            _ = new RemoteException(RemoteError.Undefined, "exception", new Exception("bang"));
-            _ = new RemoteException(RemoteError.Undefined, new Exception("bang"), true);
-            var e = new RemoteException(RemoteError.AccessControl, "exception", new Exception("bang"), true);
+            _ = new RemoteException(Guid.Empty, RemoteError.Undefined, true);
```

Comment:
Maybe we should provide a constructor without `Guid`?

Reply:
The exception should always have a `memberId` which is a `Guid` but here we are testing (for code coverage) all constructors, so we just pass `Guid.Empty`. If we replaces these lines with a constructor that does not require a `Guid`... we end up not covering the constructors that require a `Guid` - and we want to cover them.